SwiftGit2/Repository.swift

```diff
@@ -344,13 +344,13 @@ final public class Repository {
 	// MARK: - Reference Lookups
 	
 	/// Load all the references with the given prefix (e.g. "refs/heads/")
-	public func referencesWithPrefix(_ prefix: String) -> Result<[ReferenceType], NSError> {
+	public func references(withPrefix prefix: String) -> Result<[ReferenceType], NSError> {
```

Comment:
I believe this should remain `references(withPrefix:)` as per https://github.com/SwiftGit2/SwiftGit2/pull/72#discussion_r93566169